RecoTracker/MkFit/python/input.py

```diff
@@ -90,10 +90,13 @@ def apply(process, options):
             process.options.wantSummary = True
         elif options.timing == "FastTimerService":
             process.FastTimerService.enableDQMbyPath = True
+        elif options.timing == "all":
+            process.options.wantSummary = True
+            process.FastTimerService.enableDQMbyPath = True
```

Comment:
does this actually work?
I recall that Timing service and FastTimerService have some destructive interference (they run, but the timing results are wrong)

Reply:
It runs, but I never really used it for a measurement/comparison. 
If there's any doubt about it, I'll just remove this option.

Reply:
I'd suggest to drop it because of it messing up the results.

Reply:
Removed it.